.changeset/spicy-starfishes-press.md

```diff
@@ -0,0 +1,6 @@
+---
+"@graphql-codegen/client-preset": patch
+"website": patch
+---
+
+Added configuration to allow for custom hash functions for persisted documents in the client preset
```

Comment:
Can you please provide a configuration example? Right now the changeset contains little information.